frontend/device/android/device.lua

```diff
@@ -197,6 +197,14 @@ function Device:init()
                         end)
                     end
                 end
+            elseif ev.code == C.ALOOPER_FIFO_POWER_CONNECTED then
+                local Event = require("ui/event")
+                UIManager:broadcastEvent(Event:new("Charging"))
+            elseif ev.code == C.ALOOPER_FIFO_POWER_DISCONNECTED then
+                local Event = require("ui/event")
+                UIManager:broadcastEvent(Event:new("NotCharging"))
+            else
```

Comment:
please remove the else statement. A few different reasons:

1. is not an error, just a command we don't need to handle.
2. it is already logged in https://github.com/koreader/koreader/blob/99b873ece3f9ddd368037ed490bf0cdb86fc5e36/frontend/device/android/device.lua#L137
3. the message is a bit wrong. Its value is already known (and part of `android.lua`), but we decide to ignore it, because we don't need that.

The debug statement is enough for debugging and release builds shouldn't care about not handled commands.